QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -123,6 +123,8 @@ public static void setup() {
         Cluck.publish(QuasarHelios.testPrefix + "Clamp Limit Bottom", limitBottom);
         Cluck.publish(QuasarHelios.testPrefix + "Clamp Motor Speed", speedControl);
         Cluck.publish(QuasarHelios.testPrefix + "Clamp PID Output", (FloatInput) pid);
+        
+        Cluck.publish("Enter Teleop", Igneous.startTele);
```

Comment:
Why do we need this published? The drivers station tells you when you are in Teleop.